practice/timezone/timezone/views.py

```diff
@@ -0,0 +1,81 @@
+"""timezone Views."""
+
+from . import logic
+import arrow
+
+from django.http import HttpResponse
+
+
+def get_qualified_tz(latlng_string):
+    r"""Check the requested lat, lng for validity.
+
+    >>> get_qualified_tz('45,45')
+    'Europe/Moscow'
+    >>> get_qualified_tz('4545')
+    Traceback (most recent call last):
+    ...
+    KeyError: 'Need two values'
+    >>> get_qualified_tz('45,-125')
+    Traceback (most recent call last):
+    ...
+    OSError: Time Zone does not exist there.
+    """
+    lat_lng_list = latlng_string.split(',')
+    if len(lat_lng_list) != 2:
+        raise KeyError('Need two values')
+    tz_at_latlng = logic.get_tz(lat_lng_list)
```

Comment:
This function also combines two distinct and testable steps into a single step. The process of parsing a `lat,lng` string into two floats is distinct and separately testable from getting the time zone. Daisy chaining function calls makes it harder to test parts in isolation. The view functions themselves can call each of those steps.